packages/node_modules/pouchdb/src/index.js

```diff
@@ -1,2 +1,2 @@
-import PouchDB from 'pouchdb-node';
+import PouchDB from './pouchdb';
```

Comment:
made this two separate files because `"browser"` is funky and this is the only way I could get it to work when you're importing from subfolders within an external package.